src/test/java/wordsWithSpaces/WordsWithSpacesTest.java

```diff
@@ -0,0 +1,51 @@
+package wordsWithSpaces;
```

Comment:
btw: good practice to have only low-level packages

Reply:
what is it?

Reply:
I mean have `package wordswithspaces` instead of `package wordsWithSpaces`

Reply:
updated =)